nautobot_floor_plan/tests/test_integration.py

```diff
@@ -0,0 +1,70 @@
+"""Selenium tests for nautobot_floor_plan."""
+
+import unittest
+
+from django.urls import reverse
+from nautobot.core.testing.integration import SeleniumTestCase
+from selenium.webdriver.common.by import By
+from selenium.webdriver.support.ui import Select
+from selenium.webdriver.support.wait import WebDriverWait
+
+from nautobot_floor_plan.tests.fixtures import create_prerequisites
+from nautobot_floor_plan.tests.utils import is_nautobot_version_less_than
+
+
+@unittest.skipIf(is_nautobot_version_less_than("2.3.1"), "Nautobot version is less than 2.3.1")
+class RangeTileTestCase(SeleniumTestCase):
+    """Validations against floor plan custom label workflows."""
+
+    def setUp(self):
+        """Login and navigate to the floor plan custom label section."""
+        super().setUp()
+        self.pre_reqs = create_prerequisites()
```

Comment:
self.pre_reqs is assigned but never used.

Reply:
Ack, removed in latest commit